cli/azd/cmd/infra_create.go

```diff
@@ -81,7 +80,7 @@ func (ica *infraCreateAction) Run(ctx context.Context, cmd *cobra.Command, args
 
 	// Copy the parameter template file to the environment working directory and do substitutions.
 	parametersPath := azdCtx.BicepParametersTemplateFilePath(rootModule)
-	parametersBytes, err := ioutil.ReadFile(parametersPath)
+	parametersBytes, err := os.ReadFile(parametersPath)
```

Comment:
curious on why this change, from ioutil to os, is part of the PR??

Reply:
I would suggest creating a new issue about the lint error and a PR to fix that one, instead of using the same PR.
A fix for a specific issue deserves its own PR and git history entry, specially when we do squash-commit for merging PRs,
which means that there will be just one commit for your new feature to `redact access token`

No need to change or do it on this one, but keep in mind in case there's a case like this in the future.